src/FreeAge/common/player.hpp

```diff
@@ -50,18 +53,20 @@ struct PlayerStats {
   void ResearchCompleted(/* TODO */) {};
 
   /// The population of the units that are currently being produced.
-  /// TODO: implement on the client, now only used by the server
+  /// TODO: Implement on the client, now only used by the server. 
+  /// TODO: Encapsulate and use the PopulationCounter
   int populationInProduction = 0;
 
   // getters
 
-  inline int GetPopulationCount() const { return populationCount; }
-  inline int GetAvailablePopulationSpace() const { return availablePopulationSpace; }
+  inline int GetPopulationDemand() const { return doubledPopulationDemand / 2; }
```

Comment:
I renamed from Count to Demand because the `PopulationCount` could be confused with `AliveUnitCount` (which could be impl in the future)